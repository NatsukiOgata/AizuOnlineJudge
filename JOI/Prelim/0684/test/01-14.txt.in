100 20
21 80 18 32 82 2 33 77 95 91 87 10 65 30 100 88 71 75 82 58 42 87 62 51 61 11 58 5 69 87 6 34 67 27 85 32 12 23 76 94 2 42 58 79 29 27 11 70 17 12 64 13 81 99 10 17 98 48 65 77 49 97 64 71 9 17 32 10 43 94 70 83 49 55 41 23 18 78 17 71 50 28 7 57 8 59 35 28 91 34 84 16 95 54 11 25 70 43 92 22
70 33 29 50 23 8 19 80 78 44 67 60 89 28 56 49 93 27 11 39
