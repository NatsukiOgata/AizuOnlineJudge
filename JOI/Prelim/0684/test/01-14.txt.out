8
11
23
27
28
29
33
49
50
67
70
78
80
